src/components/GradeLevelFilter.jsx

```diff
@@ -10,40 +10,38 @@ const useStyles = makeStyles((theme) => ({
   },
   popOverWindow: {
     minWidth: '200px',
+    fontSize: '15px',
+    paddingTop: theme.spacing(1),
+    paddingLeft: theme.spacing(1),
+    paddingBottom: theme.spacing(1),
   },
 }));
 
-const gradeList = {
-  12: {
-    value: 'Highschool Senior',
-    isChecked: false,
-  },
-  13: {
-    value: 'College Freshman',
-    isChecked: false,
-  },
-  14: {
-    value: 'College Sophomore',
-    isChecked: false,
-  },
-  15: {
-    value: 'College Junior',
-    isChecked: false,
-  },
-  16: {
-    value: 'College Senior',
-    isChecked: false,
-  },
-};
+const gradeItems = [
+  '12th - High School',
+  'College Freshman',
+  'College Sophomore',
+  'College Junior',
+  'College Senior',
+  '5th Year',
+  'Post Grad',
+];
 
 export default function GradeLevelFilter() {
   const classes = useStyles();
   const [anchorEl, setAnchorEl] = useState(null);
   const handleClick = (event) => setAnchorEl(event.currentTarget);
+  const [selectedGrades, setSelectedGrades] = useState(new Set());
 
-  function updateGradeSelection(value) {
-    gradeList[value].isChecked = !gradeList[value].isChecked;
-    setAnchorEl(false);
+  function updateGradeSelection(gradeSelected) {
```

Comment:
Nits on naming:
- `update` is a bit generic for what could more precisely be described as [toggling](https://www.google.com/search?q=define+toggle)
- The `Grade` in `updateGradeSelection` might be a bit too specific considering how the component is named `GradeLevelFilter` and there are no other functions here on toggling selection.
- The `Selected` in `gradeSelected` is a bit redundant considering the name of this function.

You might consider renaming this function something like `toggleSelection`, `toggleSelected`, `toggleSelectedState`, etc. and the parameter `grade`.

Reply:
Ok yea does make sense, I do need to better with my naming conventions.